config/environment.js

```diff
@@ -46,6 +46,17 @@ module.exports = function(environment) {
             },
         ],
         FB_APP_ID: process.env.FB_APP_ID,
+        microfeedback: {
+            url: process.env.MICROFEEDBACK_URL,
```

Comment:
This is not needed, send the desired config over to @binoculars, we can inject it at deployment time.